src/registry_schemas/schemas/business.json

```diff
@@ -78,6 +78,11 @@
           "title": "Warning message",
           "examples": ["A minimum of 3 directors is required"]
         },
+        "type": {
```

Comment:
i'm not sure if "type" is a reserved word.  Can we use `warningType` instead of `type` here.

Reply:
I changed `type` to `warningType` in my latest commit. I have a question. If I change this to `warningType` in business schema, do I need to do the change in lear?

Reply:
yes